charter.md

```diff
@@ -85,15 +93,121 @@ are replacing, regardless of the length of that remainder.
 
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
+  of no confidence
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
+another current member.
+
+Once a vote of no confidence has been called, the committee will notify the
+community through the following channels:
+
+* the [community mailing list][dev-list]
+* the [Steering Committee public mailing list][steering-public-list]
+
+This notification will include:
+
+* a link to the aforementioned GitHub issue
+* a short statement from non-affected, seated committee members providing
+  context on the reason for the vote
+
+There will be a period of two weeks for members of the community to reach
+out to Steering Committee members to provide feedback, which can include
+non-binding votes.
```

Comment:
All for feedback (privately or publically), but I would suggest removal of the non-binding votes piece, as this could quickly turn into brigading. I would say that +1s or -1s with no other content aren't particularly valuable in this particular case.

Reply:
good point. agree.

Reply:
agree with removing the last clause completely. This sentence has gotten scrambled anyway, at the moment it appears to say community members can make non-binding votes, which makes no sense.

Reply:
Fixed in https://github.com/kubernetes/steering/pull/248/commits/c12de21656423b8840575b1254591eb293b58715.